bodhi/client/__init__.py

```diff
@@ -105,6 +107,31 @@ def _warn_if_url_and_staging_set(ctx, param, value):
                  help='Go to page number')]
 
 
+# Common releases options
+release_options = [
+    click.option('--username'),
+    click.option('--password', prompt=True, hide_input=True),
+    click.option('--name', help='Release name (eg: F20)'),
+    click.option('--long-name', help='Long release name (eg: "Fedora 20")'),
+    click.option('--id-prefix', help='Release prefix (eg: FEDORA)'),
+    click.option('--version', help='Release version number (eg: 20)'),
+    click.option('--branch', help='Git branch name (eg: f20)'),
+    click.option('--dist-tag', help='Koji dist tag (eg: f20)'),
+    click.option('--stable-tag', help='Koji stable tag (eg: f20-updates)'),
+    click.option('--testing-tag',
+                 help='Koji testing tag (eg: f20-updates-testing)'),
+    click.option('--candidate-tag',
+                 help='Koji candidate tag (eg: f20-updates-candidate)'),
+    click.option('--pending-stable-tag',
+                 help='Koji pending tag (eg: f20-updates-pending)'),
+    click.option('--pending-testing-tag',
+                 help='Koji pending testing tag (eg: f20-updates-testing-testing)'),
+    click.option('--override-tag', help='Koji override tag (eg: f20-override)'),
+    click.option('--state', type=click.Choice(['disabled', 'pending', 'current',
+                                               'archived']),
+                 help='The state of the release')]
```

Comment:
you could add the `url_option` here instead of using the decorator on each release command. 

See as an example  https://github.com/fedora-infra/bodhi/blob/develop/bodhi/client/__init__.py#L95

Reply:
I think it would not hurt to add the `staging_option` too